adr/cli.py

```diff
@@ -100,7 +100,9 @@ def run_recipe(recipe, args, fmt='table'):
     mod = importlib.import_module(modname, package='adr')
     output = mod.run(args)
 
-    if isinstance(fmt, string_types):
+    if output == 'adr_error_msg':
+        return 'Error: the queried data does not exist. (ActiveData didn\'t return any data.)'
```

Comment:
nit: You can just make this message "ActiveData didn't return any data for the specified query"

Reply:
Oh hey! I figured out where this "missing" comment box went...